vfox-web/src/app/views/static-page/about.component.html

```diff
@@ -0,0 +1,30 @@
+<div class="container">
```

Comment:
'About Us' looks great to me. Few more suggestions:
1) Clicking on 'Dashboard' could go to 'Select Services' for now. We don't have contents for a 'blank' page.
2) Shouldn't we have a 'Home' page? There's no way to go to home unless I 'SignOut'. If user is already signed in show the 'Dashboard' link; otherwise don't show it.
3) Logos seem to have disappeared from the pages such as 'Tax Mitigation' etc. Let me see if this is a problem on my end.